ext/AvizExt/viz/sensitivity.jl

```diff
@@ -310,7 +310,11 @@ function ADRIA.viz.outcome_map!(g::Union{GridLayout,GridPosition}, rs::ResultSet
         for c in 1:n_cols
             f_name = Symbol(factors[curr])
             f_vals = rs.inputs[:, f_name]
-            fv_s = round.(quantile(f_vals, b_slices), digits=2)
+            if all(mod.(f_vals, 1.0) .== 0.0)
+                fv_s = sort(unique(f_vals))
```

Comment:
I'm not sure if I fully understand what's being done here - my guess is you're trying to identify factors that only use whole numbers?

If so, the model specification (which is accessible here via `rs.model_spec`) holds information on which factors are categorical/discrete (i.e., `ptype` is integer). Looping through all factor values to check their modulo/modulus is not very efficient for this purpose.

I'm also not sure of the implications for factors that aren't categorical but do use discrete values, such as number of corals to be deployed.

The above aside, when code is a little cryptic best practice is to write a comment explaining the why of the code.

Reply:
Does the sorting mess up the ordering between x and y?